cache-caffeine/src/main/java/io/micronaut/cache/caffeine/configuration/DefaultCacheConfiguration.java

```diff
@@ -27,7 +27,8 @@
  * @since 1.0.2
  */
 @EachProperty(CacheConfiguration.PREFIX)
-public class DefaultCacheConfiguration extends CacheConfiguration {
+public class DefaultCacheConfiguration extends CaffeineCacheConfiguration {
```

Comment:
this class is public, moving it another package breaks binary compatibility. To accept this change we would have to bump the major version. Alternatively you could consider not moving this class to a different package.

Reply:
Done